backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationResolver.java

```diff
@@ -53,4 +59,29 @@ public OrganizationResolver(OrganizationService os) {
         .map(o -> new ApiOrganization(o, List.of()))
         .collect(Collectors.toList());
   }
+
+  /**
+   * Retrieves a list of all pending organizations AND organization queue items
+   *
+   * @return a list of pending organizations
+   */
+  public List<ApiPendingOrganization> getPendingOrganizations() {
+    List<ApiPendingOrganization> pendingOrgsAlreadyCreated =
+        _organizationService.getOrganizations(false).stream()
+            .map(
+                org -> {
+                  ApiOrganizationAdmin adminUser =
+                      _organizationService.getAdminUserForPendingOrganization(org);
```

Comment:
Unfortunately, this won't be okay to call here.  It will probably make thousands of calls to Okta in prod.  It will probably be necessary to just leave the admin information off of the list for orgs that were created the old way and only show it for the queue items.

At some point soon, we can just stop showing the old method orgs and then it won't be an issue anymore.